### PR TITLE
item: display item notes

### PR DESCRIPTION
**Pull request by zannkukai**
Replaces the sticky icon by the note content into the document detailed
view.

Closes rero/rero-ils#1501

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## How to test?

image

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 1]
```diff
@@ -14,65 +14,59 @@
...
-            title="{{ 'Delete' | translate}}"
-            (click)="delete(item.metadata.pid)"
-            name="delete">
-        <i class="fa fa-trash" ></i>
-    </button>
-    <ng-template #deleteInfo>
-      <button type="button" class="btn btn-sm btn-outline-danger ml-1 disabled"
+      <ng-template #notRequest>
+        <button type="buttton" class="btn btn-sm btn-outline-primary disabled"
+                title="{{ 'Item request' | translate}}"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:36]: Missing space before `}`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 2]
```diff
@@ -14,65 +14,59 @@
...
+                [popover]="tolTemplate" triggers="mouseenter:mouseleave"
+                name="request">
+          <i class="fa fa-shopping-basket" aria-hidden="true"></i>
+        </button>
+        <ng-template #tolTemplate><div [innerHtml]="cannotRequestInfoMessage | nl2br"></div></ng-template>
+      </ng-template>
+      <button *ngIf="permissions.update && permissions.update.can"
+              type="button" class="btn btn-sm btn-outline-primary ml-1"
+              title="{{ 'Edit' | translate}}"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:45]: Missing space before `}`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 3]
```diff
@@ -14,65 +14,59 @@
...
+              [routerLink]="['/records', 'items', 'edit', item.metadata.pid]"
+              name="edit">
+        <i class="fa fa-pencil"></i>
+      </button>
+      <button *ngIf="permissions.delete && permissions.delete.can; else deleteInfo"
+              type="button" class="btn btn-outline-danger btn-sm ml-1"
               title="{{ 'Delete' | translate}}"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:52]: Missing space before `}`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 4]
```diff
@@ -14,65 +14,59 @@
...
-              [popover]="tolTemplate" triggers="mouseenter:mouseleave"
+              (click)="delete(item.metadata.pid)"
               name="delete">
-        <i class="fa fa-trash"></i>
+          <i class="fa fa-trash" ></i>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:55]: Remove space before `>`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 5]
```diff
@@ -14,65 +14,59 @@
...
       </button>
-      <ng-template #tolTemplate><div [innerHtml]="deleteInfoMessage | nl2br"></div></ng-template>
-    </ng-template>
+      <ng-template #deleteInfo>
+        <button type="button" class="btn btn-sm btn-outline-danger ml-1 disabled"
+                title="{{ 'Delete' | translate}}"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:59]: Missing space before `}`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/holding-item-note/holding-item-note.component.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, Input } from '@angular/core';
+import { ItemNote } from 'projects/admin/src/app/class/items';
+
+@Component({
+  selector: 'admin-holding-item-note',
+  template: `
+    <div class="row" *ngIf="note">
+      <div class="col-sm-4 col-md-3 font-weight-bold label-title pl-5" translate>{{ note.type }}</div>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/holding-item-note/holding-item-note.component.ts:24]: `<div class="col-sm-4 col-md-3 font-weight-bold label-title pl-5">{{ note.type | translate }}</div>`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html [hunk 7]
```diff
@@ -14,60 +14,57 @@
...
-    <ng-template #notRequest>
-      <button type="buttton" class="btn btn-sm btn-outline-primary disabled"
+<div class="container px-0 mt-1 mb-2" *ngIf="item && permissions && item.metadata.issue.status !== itemIssueStatus.DELETED">
+  <div class="row">  <!-- First row :: item detail -->
+    <div class="col-sm-3">
+      <a [routerLink]="['/records', 'items', 'detail', item.metadata.pid]" name="barcode">
+        {{ item.metadata.barcode }}
+      </a>
+    </div>
+    <div class="col-sm-2" name="status" translate>{{ item.metadata.status }}</div>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html:24]: `<div class="col-sm-2" name="status">{{ item.metadata.status | translate }}</div>`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html [hunk 8]
```diff
@@ -14,60 +14,57 @@
...
-            title="{{ 'Delete' | translate}}"
-            (click)="delete(item.metadata.pid)"
-            name="delete">
-      <i class="fa fa-trash" ></i>
-    </button>
-    <ng-template #deleteInfo>
-      <button type="button" class="btn btn-sm btn-outline-danger ml-1 disabled"
+      <ng-template #notRequest>
+        <button type="buttton" class="btn btn-sm btn-outline-primary disabled"
+                title="{{ 'Item request' | translate}}"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html:36]: Missing space before `}`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html [hunk 9]
```diff
@@ -14,60 +14,57 @@
...
+        <i class="fa fa-pencil"></i>
+      </button>
+      <button *ngIf="permissions.delete && permissions.delete.can; else deleteInfo"
+              type="button" class="btn btn-outline-danger btn-sm ml-1"
               title="{{ 'Delete' | translate}}"
-              [popover]="tolTemplate" triggers="mouseenter:mouseleave"
+              (click)="delete(item.metadata.pid)"
               name="delete">
-        <i class="fa fa-trash"></i>
+        <i class="fa fa-trash" ></i>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html:55]: Remove space before `>`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html [hunk 10]
```diff
@@ -14,60 +14,57 @@
...
       </button>
-      <ng-template #tolTemplate><div [innerHtml]="deleteInfoMessage | nl2br"></div></ng-template>
-    </ng-template>
+      <ng-template #deleteInfo>
+        <button type="button" class="btn btn-sm btn-outline-danger ml-1 disabled"
+                title="{{ 'Delete' | translate}}"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/serial-holding-item/serial-holding-item.component.html:59]: Missing space before `}`

**Review by Garfield-fr — APPROVED** [review]

**Review by iGormilhit — COMMENTED** [review]
For the commit message title, I think `items` is better than `item`.

Otherwise, the commit message is approved.

**Review by benerken — APPROVED** [review]

**Pull request merged**